test/integration/verify/controls/azure_container_groups.rb

```diff
@@ -0,0 +1,15 @@
+control 'check container groups' do
+  describe azure_container_groups do
```

Comment:
After the control block, we should add the impact and the description. This is how we do in was. Do we have to do the same?

Reply:
It is not mandatory..